### PR TITLE
Pcapng comment option

### PR DESCRIPTION
**Pull request by guedou**
This PR adds the ability to read and write pcapng comments.

### CODE REVIEW

File: scapy/utils.py [hunk 1]
```diff
@@ -1732,7 +1755,8 @@ def _write_packet(self,
                       sec=None,  # type: Optional[float]
                       usec=None,  # type: Optional[int]
                       caplen=None,  # type: Optional[int]
-                      wirelen=None  # type: Optional[int]
+                      wirelen=None,  # type: Optional[int]
+                      comment=None  # type: Optional[bytes]
```
Comment (gpotter2) [inline scapy/utils.py:1759]: Isn't it kinda weird to have those arguments in `recv()` ?

File: scapy/utils.py [hunk 2]
```diff
@@ -2043,13 +2074,21 @@ def _get_time(self,
 
         return sec, usec  # type: ignore
 
-    def build_block(self, block_type, block_body):
-        # type: (bytes, bytes) -> bytes
+    def _add_padding(self, raw_data):
+        # type: (bytes) -> bytes
+        if len(raw_data) % 4 != 0:
+            padding = b"\x00" * (4 - len(raw_data) % 4)
+            raw_data += padding
```
Comment (gpotter2) [inline scapy/utils.py:2081]:
```suggestion
        raw_data += ((-len(raw_data)) % 4) * b"\x00"
```
Has the minimum amount of operations

File: scapy/utils.py [hunk 3]
```diff
@@ -1813,10 +1837,16 @@ def write_packet(self,
         if wirelen is None:
             wirelen = caplen
 
+        if hasattr(packet, "comment"):
+            comment = packet.comment  # type: ignore
+        else:
+            comment = None
```
Comment (gpotter2) [inline scapy/utils.py:1843]:
```
comment = getattr(packet, "comment", None)
```

File: scapy/utils.py [hunk 4]
```diff
@@ -1575,13 +1583,25 @@ def _read_block_epb(self, block, size):
             warning("PcapNg: EPB is too small %d/20 !" % len(block))
             raise EOFError
 
+        # Compute the options offset taking padding into account
+        if caplen % 4:
+            opt_offset = 20 + caplen + (4 - (caplen % 4))
```
Comment (gpotter2) [inline scapy/utils.py:1588]: You can do `+ (-caplen) % 4`

**Review by gpotter2 — COMMENTED** [review]
Cool feature !

File: scapy/utils.py (hunk 1 above)
Comment (guedou) [reply scapy/utils.py:1759]: What do you mean?

**Comment by guedou** [comment]
Thanks @gpotter2! I applied your comments.

File: scapy/utils.py (hunk 1 above)
Comment (gpotter2) [reply scapy/utils.py:1759]: Sorry I misread the git diff :sweat_smile:

**Review by gpotter2 — APPROVED** [review]

**Pull request merged**